engine/Default/forces_drop_processing.php

```diff
@@ -167,7 +167,7 @@
 		$scout_drones_message .= 's';
 
 	// now compile it together
-	$message = $player->getPlayerName().' has ' . $mines_message;
+	$message = 'I have ' . $mines_message;
```

Comment:
This one's a little odd, whilst it makes sense from a grammatical point of view to use "I have" as the message is sent from the player but from a gameplay/information sense I think it's nice to have the player name repeated as part of the message rather than having to look outside of the main message for that info

Reply:
Well, it follows the pattern of the alliance join/leave messages, which are in first person. And if we're making a point that it's a message sent by your ally (especially if scout messages end up being sent from a "Scout Drone" rather than the pinging player), then third person looks odd. When you receive an e-mail from someone, it's hopefully not in third person. :D 

But I'll take your preference into consideration as I figure out what to do about the scout message grouping issue. Depending on the solution I come up with (if any), this may be a moot point. I'll ask you about it again once this PR feels more complete.

Reply:
Yeah, it's not a huge deal to change it, I just noticed it as I used to use that whilst scanning through messages, but I don't know whether that's a thing that helps others as well or not